pocketimdb/movies/paginations.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+
+DEFAULT_PAGE = 1
+
+class CustomPageNumberPagination(PageNumberPagination):
+    page = DEFAULT_PAGE
+    page_size = 10
+
+    def get_paginated_response(self, data):
+        return Response({
+            'total': self.page.paginator.count,
+            'page': int(self.request.GET.get('page', DEFAULT_PAGE)),
+            'page_size': self.page_size,
```

Comment:
ovo nam ni ne treba, jer i ako stavis ovako, psotoji mogucnost da na nekoj strnanici na kraju nema taj broj filmova i onda ces ovdje da vracas 10, a u stvari zadnja stranica ima samo 2 filma

Reply:
U sustini sam to vracao da ne bih na FE zakucavao da je velicina stranice 10, vec taj info vracam kroz ovaj response u slucaju da se npr promeni taj broj u nekom trenutku na BE.

Reply:
jasno, ali kao sto reko, u jednom momentu ces vratiti za jednu stranicu za koju si trazio 10 filmova samo 3 jer ukupno imas 33 filma na stanju i onda ovaj podatak nece biti validan

Reply:
Ne moram ni slati taj podatak onda, mogu staviti na FE da je page size 10 ili da ga racunam kao total / total_pages, a ta dva podatka svakako saljem sa BE svakako. Ispada da podatak nije validan ako se tako gleda da taj podatak vraca broj elemenata koje je vratio. A ja njime nisam na to mislio, vec generalno velicinu stranice

Reply:
Sad sam shvatio da mi page_size ni ne treba uopste..